app/controllers/responses_controller.rb

```diff
@@ -8,4 +10,19 @@ def show
     response = Response.find params[:id]
     render json: ResponseSerializer.new(response).as_json
   end
+
+  def update
+    response_finished = FinishResponse.run response_uuid: params[:id]
+    if response_finished.valid?
+      head :ok
+    else
```

Comment:
Je kan als result van de FinishResponse de symbol meegeven wat de status moet zijn. De interaction heeft zijn werk kunnen voltooien, aangezien het werk al voltooid is. De interaction afbreken met een exceptie daarvoor lijkt mij wat overdreven.